[devops] Fixing issue "make[1]: *** No rule to make target `clean'.  Stop." in BVT

<!-- Please describe your issue in English. -->

#### Can be reproduced ?
yes

#### Steps:
make[1]: *** No rule to make target `clean'.  Stop.

#### Expected behavior:
build matrixone should be successful

#### Actual behavior:
failed to build

#### Environment:
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:

#### Configuration file:


#### Additional context:
- Error message from client:
- Server log:
- Other information:

--- a/optools/run_bvt.sh
+++ b/optools/run_bvt.sh
@@ -149,6 +149,7 @@ function make_one(){
 }
 
 function patch_one(){
+	local current_pwd=$(pwd)
     cd $G_STAGE
 	if [[ -f patch_one.sh ]]; then
 		logger "INF" "Patch issue fix"
@@ -156,7 +157,7 @@ function patch_one(){
 	else
 		logger "INF" "Not found patch"
 	fi
-
+	cd $current_pwd
 }
 
 function run_bvt(){

